go-packages/meep-gis-asset-mgr/asset-mgr.go

```diff
@@ -245,44 +252,79 @@ func (pc *Connector) CreateDb(name string) (err error) {
-		path_fraction   decimal(10,6)         	NOT NULL DEFAULT '0.000000',
-		poa				varchar(100)			NOT NULL DEFAULT '',
-		poa_distance    decimal(10,3)         	NOT NULL DEFAULT '0.000',
-		poa_in_range	varchar(100)[]			NOT NULL DEFAULT array[]::varchar[],
-		poa_type_prio	varchar(20)[]			NOT NULL DEFAULT array[]::varchar[],
-		connected		boolean					NOT NULL DEFAULT 'false',
-		start_time		timestamptz 			NOT NULL DEFAULT now()
+	_, err = am.db.Exec(`CREATE TABLE ` + UeTable + ` (
+		id              varchar(36)             NOT NULL,
+		name            varchar(100)            NOT NULL UNIQUE,
+		position        geometry(POINT,4326)    NOT NULL,
+		path            geometry(LINESTRING,4326),
+		path_mode       varchar(20)             NOT NULL DEFAULT 'LOOP',
+		path_velocity   decimal(10,3)           NOT NULL DEFAULT '0.000',
+		path_length     decimal(10,3)           NOT NULL DEFAULT '0.000',
+		path_increment  decimal(10,6)           NOT NULL DEFAULT '0.000000',
+		path_fraction   decimal(10,6)           NOT NULL DEFAULT '0.000000',
+		poa             varchar(100)            NOT NULL DEFAULT '',
+		poa_distance    decimal(10,3)           NOT NULL DEFAULT '0.000',
+		poa_in_range    varchar(100)[]          NOT NULL DEFAULT array[]::varchar[],
+		poa_type_prio   varchar(20)[]           NOT NULL DEFAULT array[]::varchar[],
+		connected       boolean                 NOT NULL DEFAULT 'false',
+		start_time      timestamptz             NOT NULL DEFAULT now(),
+		PRIMARY KEY (id)
 	)`)
 	if err != nil {
 		log.Error(err.Error())
 		return err
 	}
 	log.Info("Created UE table: ", UeTable)
 
+	// UE Measurements Table
+	_, err = am.db.Exec(`CREATE TABLE ` + UeMeasurementTable + ` (
+		id              varchar(36)             NOT NULL,
+		ue              varchar(36)             NOT NULL,
+		poa             varchar(100)            NOT NULL DEFAULT '',
+		type            varchar(20)             NOT NULL DEFAULT '',
+		radius          decimal(10,1)           NOT NULL DEFAULT '0.0',
+		distance        decimal(10,3)           NOT NULL DEFAULT '0.000',
+		in_range        boolean                 NOT NULL DEFAULT 'false',
+		rssi            decimal(10,3)           NOT NULL DEFAULT '0.000',
+		rsrp            decimal(10,1)           NOT NULL DEFAULT '0.0',
+		rsrq            decimal(10,1)           NOT NULL DEFAULT '0.0',
+		PRIMARY KEY (id),
+		FOREIGN KEY (ue) REFERENCES ` + UeTable + `(name) ON DELETE CASCADE
+	)`)
+	if err != nil {
+		log.Error(err.Error())
+		return err
+	}
+	log.Info("Created UE Measurements table: ", UeMeasurementTable)
+
 	// POA Table
-	_, err = pc.db.Exec(`CREATE TABLE ` + PoaTable + ` (
-		id 				varchar(36) 			NOT NULL PRIMARY KEY,
-		name 			varchar(100) 			NOT NULL UNIQUE,
-		type 			varchar(20)				NOT NULL DEFAULT '',
-		radius			decimal(10,1) 			NOT NULL DEFAULT '0.0',
-		position		geometry(POINT,4326)	NOT NULL
+	_, err = am.db.Exec(`CREATE TABLE ` + PoaTable + ` (
+		id              varchar(36)             NOT NULL,
```

Comment:
Same comment applies with the PoAs that are referenced in the UE/UE meas. tables

Reply:
Agreed. The POAs are used in multiple tables, making the relations more complex in this case. This is definitely an improvement that could be made but warrants more time & effort to implement.